src/components/sidebar/right-sidebar/LastUpdated.tsx

```diff
@@ -17,9 +17,10 @@ export default function LastUpdated() {
               offset={10}
               position="top right"
               color="#455a64"
-              popupSize="mini"
+              tooltipHeader={"Last Updated"}
+              popupSize="small"
               size="sm"
-              tooltip={`Updated every 24 hours.`}/>
+              tooltip={'Last time the data was refreshed from the database, which gets updated every 24 hours.'}/>

```

Comment:
Did Rahul or Jordan tell us to replace this string? Personally I think it's a bit wordy, not sure people really need/care about any info beyond what the update rate is.